pyvistaqt/dialog.py

```diff
@@ -96,42 +99,44 @@ def value(self):
             float(super().value()) / self._max_int * self._value_range + self._min_value
         )
 
-    def setValue(self, value):
+    def set_value(self, value):
```

Comment:
And ones like this seem backward incompatible, which is not good

Reply:
(and also against the style of the given class -- using VTK or Qt-like naming -- which is a secondary but still relevant concern)

Reply:
```suggestion
    def setValue(self, value):
```